schemas/test/en/test_new_routing_section_dependencies_calculated_summary.json

```
@@ -0,0 +1,279 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Routing and Skipping Section Dependencies based on Calculated Summary",
+    "theme": "default",
+    "description": "A questionnaire to test routing and skipping rules, when the rule references a different section to its current section",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Hub",
+        "options": {}
+    },
+    "sections": [
+        {
+            "title": "Calculated Summary Section",
+            "summary": { "show_on_completion": true },
+            "id": "calculated-summary-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "first-question-block",
+                            "question": {
+                                "id": "first-question",
+                                "title": "How much do you spend on the following items?",
```

Maybe some explanation like:
```suggestion
                                "title": "How much do you spend on the following items?",
                                "description": [
                                    "<p>If the total is equal or over 100£ a new section will get enabled on the routing path</p>"
                                ],
```
The wording is not ideal but you will get the idea.

---

Added description 👍